lib/sanbase_web/generic_admin/ARCHITECTURE.md

```
@@ -0,0 +1,232 @@
+        resource : String
+        search_fields : list of atom
+    }
+
+    ResourceModule --> ResourceConfig : produces via resource/0
+    ResourceConfig --> FieldOverride : fields_override values
+    ResourceConfig --> BelongsToField : belongs_to_fields values
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)